service/controller/v1/resource/persistentvolume/update.go

```diff
@@ -93,13 +96,25 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateState inter
 			return microerror.Maskf(err, "failed to delete cleanup job", cleanupJob.Name)
 		}
 
+		r.logger.LogCtx(ctx, "pvc", pvc.Name, "removing pvc protection finalizer", pvc.Name)
```

Comment:
"protection finalizer" sounds strange. All finalizers are for protection. Also I'd like prevent some random keys.

```suggestion
		r.logger.LogCtx(ctx, "removing finalizer from pvc %#q", pvc.Name)
```

Reply:
It is called `pvc-protection`

Reply:
ACK. I would never realize anyway. If you want to print the finalizer name it would be good to take it from the object.